index.html

```diff
@@ -8,6 +8,69 @@
     <link href="./styles.css" rel="stylesheet" >
```

Comment:
there is a duplicate fb logo in the images folder . Delete it. 

rename playstore.svg to appleStore

change tele.svg to telegram.svg